lib/policy.py

```diff
@@ -33,7 +33,16 @@
 
 DEFINITIONS = None
 DEFAULT_DEFINITIONS = './def'
-ACTIONS = set(('accept', 'deny', 'reject', 'next', 'reject-with-tcp-rst'))
+ACTIONS = set(('accept', 'count', 'deny', 'reject', 'next',
+               'reject-with-tcp-rst'))
```

Comment:
This is to resolve a conflict where 'count' was just added via:
https://github.com/google/capirca/commit/74e16b4e378ff96a29d2d09ff91b0858c8bc921e#diff-1b56210f6b46f87bcd833cb0cac39c26